index.html

```diff
@@ -12,6 +12,16 @@
       <div class="search">
         <input class="search--input" type="text" placeholder="What emoji are you looking for?">
       </div>
+      <ul class="emojis">
+        <li class="emojis--item"><a href="#" title="smiling">😀</a></li>
+        <li class="emojis--item"><a href="#" title="laughter">😂</a></li>
+        <li class="emojis--item"><a href="#" title="">😃</a></li>
+        <li class="emojis--item"><a href="#" title="">😄</a></li>
+        <li class="emojis--item"><a href="#" title="">😅</a></li>
+        <li class="emojis--item"><a href="#" title="">😆</a></li>
+        <li class="emojis--item"><a href="#" title="">😇</a></li>
+        <li class="emojis--item"><a href="#" title="">😇</a></li>
+      </ul>
```

Comment:
Here's some documentation about the ordering of emojis conceptually - easy to follow: http://www.unicode.org/emoji/charts/emoji-ordering.html